.circleci/config.yml

```diff
@@ -0,0 +1,30 @@
+version: 2
+jobs:
+    build:
+        docker:
+            - image: circleci/python:3.6.1
+        steps:
+            - checkout
+            - run:
+                command: |
+                  wget -q http://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh -O ~/miniconda.sh;
+                  chmod +x ~/miniconda.sh;
+                  ~/miniconda.sh -b -p ~/miniconda;
+                  echo "export PATH=~/miniconda/bin:$PATH" >> $BASH_ENV;
+            - run:
+                command: |
+                  conda update --yes --quiet conda;
+                  conda create --yes --name testenv python=3
+                  source activate testenv;
+                  conda install --yes --quiet numpy scipy matplotlib pandas
+                  pip install flake8 pytest nose coverage coveralls
+                  git clone --depth 1 https://github.com/mne-tools/mne-python.git -b maint/0.15
+                  cd mne-python
+                  pip install --no-deps -e .
+                  cd ../
+                  python setup.py develop
+                  npm install -g https://github.com/INCF/bids-validator.git#0.26.0
```

Comment:
why is this necessary?

Reply:
I would use Circle CI only for building examples, not for tests ...

Reply:
okay, fair enough. Travis is enough, I'll skip the testing :-)